stdlib/LinearAlgebra/src/dense.jl

```diff
@@ -907,6 +907,25 @@ end
 sqrt(A::AdjointAbsMat) = adjoint(sqrt(parent(A)))
 sqrt(A::TransposeAbsMat) = transpose(sqrt(parent(A)))
 
+# Cube root of a complex-valued matrix
+cbrt(A::AbstractMatrix) = A^(1//3)
```

Comment:
I don't think we want to have this method.  It is an inconsistent branch cut with the real case, and we don't support `cbrt` for scalars either: https://github.com/JuliaLang/julia/issues/36534

Reply:
Incorporated

Reply:
The method is still present?

Reply:
Unfortunately, I think I marked it too quickly. The change is actually on my PC and will be pushed post testing. In the meantime, I will incorporate the other changes. Thank you!